### ISSUE TITLE
Hanging braces on methods and classes

### ISSUE THREAD

**Issue by methnen** [WordPress-Coding-Standards/WordPress-Coding-Standards#347]
Running code that has non hanging braces on methods and classes is passing the WordPress Coding Standards sniffer.  

I'm pretty sure that it shouldn't be unless that's not a hard standard possibly?

i.e. this will pass:

```php
class Class_Name 
{
	public function function_name()
	{
		
	} // END __construct
}
```

Though I'm pretty sure this is the actual standard:

```php
class Class_Name  {
	public function function_name() {
		
	} // END __construct
}
```

**Comment by westonruter** [comment]
You're right. Opening braces should be on the same line as the opening matter. There should actually be an existing sniff that implements this which we can reference, but I just haven't looked hard enough for it. Here you go! https://github.com/squizlabs/PHP_CodeSniffer/tree/master/CodeSniffer/Standards

**Comment by GaryJones** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/Generic/Sniffs/Functions/OpeningFunctionBraceKernighanRitchieSniff.php

Can't see an equivalent check for classes, interfaces, traits etc.

**Comment by JDGrimes** [comment]
> Can't see an equivalent check for classes, interfaces, traits etc.

I can't find one either. Maybe @gsherwood can tell us if there are any?

**Comment by gsherwood** [comment]
> I can't find one either. Maybe @gsherwood can tell us if there are any?

Only this kind of thing: https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PEAR/Sniffs/Classes/ClassDeclarationSniff.php

Feature request to bring something like this into the Generic sniff set is here, but I'm not working on it at the moment: https://github.com/squizlabs/PHP_CodeSniffer/issues/532

**Comment by jrfnl** [comment]
I believe this has been implemented already for functions by now and PR #596 would take care of the class braces.

**Comment by JDGrimes** [comment]
Fixed with #596.
